stytch/shared/rbac_local.go

```diff
@@ -17,7 +17,7 @@ func PerformAuthorizationCheck(
 	}
 
 	if subjectOrgID != authorizationCheck.OrganizationID {
-		return stytcherror.NewClientLibraryError("Subject organization ID does not match ID from request")
+		return stytcherror.NewSessionAuthorizationTenancyError(subjectOrgID, authorizationCheck.OrganizationID)
```

Comment:
The constructor seems to expect authCheck.orgID first -- or maybe it's just not clear which is "expected"
The Ruby error makes this explicit:
```rb
msg = "Subject organization_id #{subject_org_id} does not match authZ request organization_id #{request_org_id}"
```